src/main/java/org/jabref/gui/collab/DatabaseChangeMonitor.java

```diff
@@ -62,10 +66,31 @@ public DatabaseChangeMonitor(BibDatabaseContext database,
             }
         });
 
+        LibraryTab.DatabaseNotification notificationPane = libraryTab.getNotificationPane();
         addListener(changes -> notificationPane.notify(
                 IconTheme.JabRefIcons.SAVE.getGraphicNode(),
                 Localization.lang("The library has been modified by another program."),
-                List.of(new Action(Localization.lang("Dismiss changes"), event -> notificationPane.hide()),
+                List.of(new Action(Localization.lang("Dismiss changes"), event -> libraryTab.getNotificationPane().hide()),
+                        new Action(Localization.lang("Accept changes"), event -> {
+                            // Perform all accepted changes
+                            NamedCompound ce = new NamedCompound(Localization.lang("Merged external changes"));
+                            for (DatabaseChangeViewModel change : changes) {
+                                if (change instanceof EntryChangeViewModel) {
+                                    // We don't have a checkbox for accept and always get the correct merged entry, the accept property in this special case only controls the radio buttons selection
+                                    change.makeChange(database, ce);
+                                } else if (change.isAccepted()) {
+                                    change.makeChange(database, ce);
```

Comment:
These both statements are equal, aren't they? Think, the `if` condition is not needed.